bowtie/_cli.py

```diff
@@ -809,13 +828,25 @@ def statistics(report: _report.Report, n: int, format: _F):
             else {}
         ),
     )
+    header = (
+        f"Dialect: {report.metadata.dialect.pretty_name}\n"
+        f"Last ran on: "
+        f"{report.metadata.started.strftime(USER_LOCALE_DT_FMT)}"
```

Comment:
Are you looking for [`%x`](https://docs.python.org/3/library/datetime.html#strftime-and-strptime-format-codes)? There's no need to touch the `locale` module directly for this.